### DISCUSSION TITLE
卷積神經網路CNN

### DISCUSSION THREAD

**Discussion by Pietra1226** [1101-GS4719-C/Python-Community-GS4719-C#762] [General]
學習python除了是因為語法相對其他簡單外，python也有很多模組可以使用。
而現今影像辨識是很熱門的主題，python也是相對於其他語言更適合做這類深度學習，神經網路的程式語言。
而Convolution Neural Network算是針對特定項目有很精準辨識的神經網路。
### 目前了解
1. 圖片網格向量展開
2. 自訂Receptive Field跟Stride
3. Convolution
4. Pooling
5. 重複3.4.(自訂，也可以不要pooling)
6. Flatten
7. 送到全連接神經網路
8. Result

以上是我基本的了解，如果有錯的話都可以糾正~~
下面是兩個幫助我大致了解這項技術的影片跟網站
[講解影片](https://www.youtube.com/watch?v=OP5HcXJg2Aw)
[概念網站](https://brohrer.mcknote.com/zh-Hant/how_machine_learning_works/how_convolutional_neural_networks_work.html)

**Comment by danwan1027** [comment]
太厲害了！感謝分享

**Comment by williamyen0623** [comment]
感謝分享！神經網路的學習資源相當珍貴

**Comment by manipulateed** [comment]
這對於想要深入深度學習的人類也很大的幫助，內容清晰深入淺出。

**Comment by evanchou1204** [comment]
之前修李秉宏老師的AI人工智慧導論，用到的AI功能就是影像辨識，只要丟一張照片進去AI就可以比對是否正確，很神奇！
